services/movie_session.py

```diff
@@ -42,3 +45,13 @@ def update_movie_session(
 
 def delete_movie_session_by_id(session_id: int) -> None:
     MovieSession.objects.get(id=session_id).delete()
+
+
+def get_taken_seats(movie_session_id: int) -> list[dict]:
+    if not MovieSession.objects.filter(id=movie_session_id).exists():
+        raise ValueError("Movie session does not exist")
+    return list(
+        MovieSession.objects.get(id=movie_session_id)
```

Comment:
How about to use MovieSession.objects.filter(id=movie_session_id).first() in the first place to avoid requesting db twice?